syft/workers/base.py

```diff
@@ -678,7 +678,7 @@ def __getitem__(self, idx):
     def clear_objects(self):
         """Removes all objects from the worker."""
 
-        self._objects = {}
+        self._objects.clear()
```

Comment:
Could you add a test for this?

Reply:
Done. 
However, Travis fails due to 1 test case failing test/federated/test_federated_client.py::test_fit[gaussian_mixture-1]

the failed assertion is -
assert loss_after < loss_before
assert tensor(0.6931, grad_fn=<NllLossBackward>) < tensor(0.6931, grad_fn=<NllLossBackward>)

The above test case passes on my local development setup though. 
Should we trigger Travis build again? 
Or, I think the assert should be changed to "assert loss_after <= loss_before" we shouldn't assert if the loss remains the same.

Reply:
https://gitlab.com/pycqa/flake8-docstrings/merge_requests/15